pydy/viz/scene.py

```diff
@@ -472,17 +472,17 @@ def display_ipython(self):
 
         self.create_static_html(silent=True)
         self._widget_dict = OrderedDict()
-        self.container = widgets.ContainerWidget()
+        self.container = widgets.Box()
         components = []
         for var, init_val in \
             zip(self.constant_variables, self.constant_values):
-            self._widget_dict[str(var)] = widgets.FloatTextWidget(value=init_val,
-                                                              description=str(var))
+            self._widget_dict[str(var)] = widgets.FloatText(value=init_val,
+                                                        description=str(var))
             components.append(self._widget_dict[str(var)])
 
-        self.button = widgets.ButtonWidget(description="Rerun Simulations")
+        self.button = widgets.Button(description="Rerun Simulations")
         def button_click(clicked):
-            self.button.add_class('disabled')
+            self.button._dom_classes = ['disabled']
```

Comment:
They suggested this way to do something they don't support http://ipython.org/ipython-doc/3/whatsnew/version3_widget_migration.html
and http://comments.gmane.org/gmane.comp.python.ipython.devel/14879
I don't think there is any reason we shouldn't. Also, we are including our custom html. Thus, thinkg which ipython support are not useful. They provide them to replace css and other properties.